custom/aaa/static/aaa/js/models/pregnant_women.js

```diff
@@ -34,7 +33,7 @@ hqDefine("aaa/js/models/pregnant_women", [
         });
 
         self.highRiskPregnancy = ko.computed(function () {
-            return self.highRiskPregnancy === 1 ? 'Yes': 'No';
+            return self.highRiskPregnancy() === 1 ? 'Yes': 'No';
```

Comment:
I think this will be either 'yes', not 1 as that's what's stored in the database
```suggestion
            return self.highRiskPregnancy() === 'yes' ? 'Yes': 'No';
```